test/ILLink.RoslynAnalyzer.Tests/AnalyzerTests.cs

```diff
@@ -23,9 +23,80 @@ class C
     int M1() => 0;
     int M2() => M1();
 }";
-			return VerifyCS.VerifyAnalyzerAsync (src,
+			return VerifyCS.VerifyAnalyzerAsync (TestRequiresWithMessageOnlyOnMethod,
 				// (8,17): warning IL2026: Calling 'System.Int32 C::M1()' which has `RequiresUnreferencedCodeAttribute` can break functionality when trimming application code. message.
-				VerifyCS.Diagnostic ().WithSpan (8, 17, 8, 21).WithArguments ("C.M1()", "message")
+				VerifyCS.Diagnostic ().WithSpan (8, 17, 8, 21).WithArguments ("C.M1()", "message", "")
+				);
+		}
+
+		[Fact]
+		public Task TestRequiresWithMessageAndUrlOnMethod ()
```

Comment:
Why do we need to rewrite these tests here?

Reply:
We don't, I just believe that it was important to have an individual test on named arguments since it exercises a specific case in the resource file, and something that test setters and getters separated from the other.